### PR TITLE
feat: add tokens for AgoraDex

### PR DESCRIPTION
**Pull request by oliviera9**

### CODE REVIEW

**Review by envin3 — CHANGES_REQUESTED** [review]
I would split the commit between actual features and tests.
Double checked decimal and they are correct.

File: src/settings/swap-assets.js [hunk 1]
```diff
@@ -365,6 +365,29 @@ const swapAssets = [
     chainId: ChainId.TelosMainnet,
     onPnetworkV2: true,
   },
+  {
+    address: 'eosio.token',
```
Comment (envin3) [inline src/settings/swap-assets.js:369]: Is this address correct?

File: src/settings/swap-assets.js [hunk 2]
```diff
@@ -407,6 +430,29 @@ const swapAssets = [
     chainId: ChainId.LibreMainnet,
     onPnetworkV2: true,
   },
+  {
+    address: 'eosio.token',
```
Comment (envin3) [inline src/settings/swap-assets.js:434]: address?

File: src/settings/swap-assets.js [hunk 3]
```diff
@@ -1039,6 +1085,52 @@ const swapAssets = [
     withBalanceDecimalsConversion: true,
     chainId: ChainId.XdaiMainnet,
   },
+  {
+    address: 'eosio.token',
```
Comment (envin3) [inline src/settings/swap-assets.js:1089]: address?

File: src/settings/swap-assets.js (hunk 1 above)
Comment (oliviera9) [reply src/settings/swap-assets.js:369]: Yes it is. For a confirmation, check PUOS_ON_ULTRA.

**Review by envin3 — APPROVED** [review]

**Comment by envin3** [comment]
v2 support dropped

**Pull request closed without merging**